lighthouse-core/lib/network-recorder.js

```diff
@@ -53,22 +130,17 @@ class NetworkRecorder extends EventEmitter {
    */
   onRequestStarted(request) {
     this.startedRequestCount++;
+    request.data._observedNodeStartTime = Date.now();
     this._records.push(request.data);
 
     const activeCount = this.activeRequestCount();
-    log.verbose('NetworkRecorder', `Request started. ${activeCount} requests in progress` +
-        ` (${this.startedRequestCount} started and ${this.finishedRequestCount} finished).`);
-
-    // If only one request in progress, emit event that we've transitioned from
-    // idle to busy.
-    if (activeCount === 1) {
-      this.emit('networkbusy');
-    }
+    log.verbose(
```

Comment:
do we want to keep these lines attached to activeRequestCount now? since our idling logic isn't related it seems better to drop the logs or have them log out the networkStatus

Reply:
yeah good point I just removed all other active request count tracking since we don't really need it, moved log to network status